### PR TITLE
Fixed vsphere load balancer classes selection

### PR DESCRIPTION
**Pull request by grolu**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```bugfix user
Fixed vSphere Load Balancer Class selection
```

### CODE REVIEW

File: frontend/src/components/NewShoot/GNewShootInfrastructureDetails.vue [hunk 1]
```diff
@@ -174,27 +174,27 @@ SPDX-License-Identifier: Apache-2.0
...
+              <v-list-item
+                v-bind="props"
+                :disabled="item.raw.disabled"
+              >
+                <template #prepend>
+                  <v-icon>
+                    {{ isLoadBalancerClassSelected(item) ? 'mdi-checkbox-marked' : 'mdi-checkbox-blank-outline' }}
+                  </v-icon>
+                </template>
+              </v-list-item>
```
Comment (holgerkoser) [inline frontend/src/components/NewShoot/GNewShootInfrastructureDetails.vue:197]:
```suggestion
              <v-list-item
                v-bind="props"
                :disabled="item.raw.disabled"
                :prepend-icon="isLoadBalancerClassSelected(item) ? 'mdi-checkbox-marked' : 'mdi-checkbox-blank-outline'"
              />
```

**Review by petersutter — APPROVED** [review]
/lgtm

**Review by holgerkoser — APPROVED** [review]
/lgtm

**Pull request merged**